en/setup/citationkeypatterns.md

```diff
@@ -152,7 +152,7 @@ To change the citation key pattern to `[authors][camel]` for all libraries witho
 
 1.  Open the preferences
 
-    <img src="../.gitbook/assets/optionspreferences (3) (2) (2) (2) (1) (3) (3) (4) (4) (5) (1) (1) (3).png" alt="Options Preferences" data-size="original">
```

Comment:
@ThiloteE do you have the time to check on this doc and update the image here? My KDE snipping tool wont allow me to take screenshots of open menus...

Reply:
can do it, but if i change it via gitbook, would that not create a conflict with this pull-request?

Reply:
Probably yes, so let's wait until this is merged